src/lib/logger.js

```diff
@@ -0,0 +1,42 @@
+import pino from "pino"
+const httpLogger = require("pino-http")({
+  base: {
+    pid: undefined,
+  },
+
+  timestamp: pino.stdTimeFunctions.isoTime,
+
+  // overwrite the req and res obejct specifying properties to keep
+  serializers: {
+    req(req) {
+      return {
+        useragent: req.headers["user-agent"],
+        sourceIP: req.remoteAddress,
+        hostname: req.headers.host,
+        port: req.headers.remotePort,
+        requestURI: req.url,
+        requestMethod: req.method,
```

Comment:
Must these use a `request` prefix? The properties seem unambiguous, and several others are named without it.

Reply:
Was following the [OWASP example](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Logging_Vocabulary_Cheat_Sheet.md#format) for naming properties but removed in https://github.com/artsy/positron/pull/3102/commits/cde7afc4a80300650ab9987299851e1d6492e480